packages/playwright-core/src/server/fetch.ts

```diff
@@ -190,9 +193,11 @@ export abstract class APIRequestContext extends SdkObject {
       maxRedirects: params.maxRedirects === 0 ? -1 : params.maxRedirects === undefined ? 20 : params.maxRedirects,
       timeout,
       deadline,
+      ...(process.env.PW_DO_NOT_USE_EXTRA_CA_CERTS ? { ca: [fs.readFileSync(process.env.PW_DO_NOT_USE_EXTRA_CA_CERTS)] } : {}),
```

Comment:
- nit: I'd add an `if` statement at line 199.5 instead.
- Should it be `PWTEST_CUSTOM_CA`? It confusing has "do not use" in the env name, and then uses the ca.

Reply:
I'd like to make the name clear that this is unsupported. Changed it to `UNSUPPORTED` instead.